app/backend/user/models.py

```
@@ -153,3 +187,10 @@ class User(
 
     def __str__(self):
         return self.email
+
+    def delete(self, *args, **kwargs):
+        """Override delete to remove profile image file"""
+        if self.profile_image:
+            if os.path.isfile(self.profile_image.path):
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)